index.html

```diff
@@ -881,7 +881,7 @@ <h3> Why do some people enjoy physical pain or become aroused by it? For Kal, th
                                 shares about his years of self-destruction and bottoming out before eventually getting
                                 sober and starting the process of healing and advocating for other survivors. </h3>
 
-                                <div class="soundcite" data-url="https://content.production.cdn.art19.com/validation=1652897236,26b0cf60-7ebb-5b17-80b5-83c2f9fff92c,P2mucaQC5tSevmOA6d3Oa_eLn50/episodes/bad019ee-9b56-4952-914c-7ffce5e74f4e/e69a9326a1f7252eda2c12a3e51fa88d4b8d697b9a50905932b32deaf53245b24a870e9860bef7b96866867b6f42858030f945803c22041f9743cdf46ad4ca18/342_When_Your_Trauma_Informs_Your_Turn_Ons.mp3%3Ft%3D1576123443%26tdtok%3DeyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImtpZCI6Ink5dEdIIn0.eyJpc3MiOiJvbW55IiwiaWF0IjoxNjMxNzQ4MzEyLCJ0ZC1yZWciOmZhbHNlLCJ0ZC1hZHMiOiJub25lIn0.mp3" data-start="1865000" data-end="5898000" data-plays="1">Excerpt of Audio</div>
+                                <div class="soundcite" data-url="https://content.production.cdn.art19.com/validation=1663449286,636972be-e441-5cca-9bb7-3a68388c064e,n-GMVSp-VRpWYv0qxXEdSpDDs9Q/episodes/bad019ee-9b56-4952-914c-7ffce5e74f4e/69cf7a2f9c510890edc3303e74f4933fd2e19e63eab0a51392df62cf8932d57323d93cfcd9a2c96369a2feb7b70455dbf8ade98e1ab154e04c3337017cd1a4ea/342_When_Your_Trauma_Informs_Your_Turn_Ons.mp3%3Ft%3D1576123443%26tdtok%3DeyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImtpZCI6Ink5dEdIIn0.eyJpc3MiOiJvbW55IiwiaWF0IjoxNjMxNzQ4MzEyLCJ0ZC1yZWciOmZhbHNlLCJ0ZC1hZHMiOiJub25lIn0.mp3" data-start="1865000" data-end="5898000" data-plays="1">Excerpt of Audio</div>
```

Comment:
You can kind of see it here, the URLs are near identical, but the `/validation=...` string/key changed.